src/components/Profile/EulaHistory/components/ListItem.tsx

```diff
@@ -40,9 +42,11 @@ const ListItem = (props: Props) => {
       </TD>
       <TD>
         <Caption13>
-          <IconWrapper onClick={downloadDocument}>
-            <DownloadIcon />
-          </IconWrapper>
+          <a href={`${DOCUMENTS_URL}/${id}`}>
```

Comment:
Тут разве не нужен атрибут download?
```suggestion
          <a href={`${DOCUMENTS_URL}/${id}`} download>
```